LayoutKit/LayoutArrangement.swift

```diff
@@ -119,7 +119,7 @@ public struct LayoutArrangement {
             return sublayout.makeSubviews(from: recycler, prepareAnimation: prepareAnimation)
         })
         // If we are preparing an animation, then we don't want to update frames or configure views.
-        if let view = layout.makeView(from: recycler) {
+        if let view = recycler.makeOrRecycleView(havingViewReuseId: layout.viewReuseId, viewProvider: layout.makeView), layout.needsView {
```

Comment:
does this mean that first the view will be made/recycled, and only after that the needsView property would be checked?